ohsome/helper.py

```diff
@@ -4,35 +4,49 @@
 """Ohsome utility functions"""
 
 import datetime
+import re
 
 import geopandas as gpd
+import numpy as np
 import pandas as pd
 
 from ohsome import OhsomeException
 
-import re
+
+def convert_arrays(params: dict) -> dict:
+    """Convert arrays to lists."""
```

Comment:
add description of `params` parameter in docstring

Reply:
solved